### PR TITLE
Sponsors page

### PR DESCRIPTION
**Pull request by postalservice14**

### CODE REVIEW

**Comment by jeremeamia** [comment]
thanks, I'll check it out.

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "gulp-ruby-sass": "^1.0.1",
     "gulp-sass": "^1.3.3",
     "gulp-size": "^1.2.1",
-    "gulp-uglify": "^1.1.0"
+    "gulp-uglify": "^1.1.0",
+    "vinyl-source-stream": "^1.1.0"
```
Comment (postalservice14) [inline package.json:25]: I had to add this to get mine to build...not sure if it was just on my machine or what...

File: resources/views/layout.blade.php [hunk 2]
```diff
@@ -37,6 +37,7 @@
                 <li><a href="./speakers">Speakers</a></li>
                 <li><a href="./talks">Talks</a></li>
                 <li><a href="./events">Events</a></li>
+                <li><a href="./sponsors">Sponsors</a></li>
```
Comment (jeremeamia) [inline resources/views/layout.blade.php]: There is a sponsors link on the right side of the navbar. Don't need this one.

**Comment by jeremeamia** [comment]
Looks good to me. I think we should still have the community sponsors tiled at the bottom like the other pages. Will have to do some jiggery pokery to get that in there without duplicating that content.

File: resources/views/layout.blade.php (hunk 2 above)
Comment (postalservice14) [inline resources/views/layout.blade.php]: Ah true, didn't see that one

**Comment by postalservice14** [comment]
"jiggery pokery" - done :-p

**Comment by postalservice14** [comment]
Addresses issue: https://github.com/pnwphp/pnwphp.com/issues/9

**Comment by jeremeamia** [comment]
Awesome. Can you change your pull request to merge into the `develop` branch instead of `master`?

**Pull request closed without merging**